scipy/sparse/tests/test_base.py

```diff
@@ -250,9 +250,9 @@ def init_class(cls):
         cls.datsp = cls.spcreator(cls.dat)
 
         # set array/matrix testing mode for this class based on the class attribute
-        # Could use spcreator._is_array except that some test classes (e.g. TextCSR)
+        # Could use isinstance(spcreator, sparray) except that some test classes (e.g. TextCSR)
         # use a method to filter warnings produced when creating the sparse object.
-        cls._is_array = cls.datsp._is_array
+        cls._is_array = isinstance(cls.datsp, sparray)
```

Comment:
@dschult, if I comment out this line, I don't get any test failures. Should I remove this?

Reply:
Yes, you should be able to remove this line and the comments before it now that we don't use the `is_array` attribute in the tests.

Reply:
Cool, thanks for checking!